assignments/src/assignment1/Assignment1.java

```diff
@@ -0,0 +1,40 @@
+package assignment1;
+
+import java.io.File;
+import java.util.Scanner;
+public class Assignment1 {
+    private void fileFinder(){
+        // address of files or folder where to search
+        File file = new File("C:\\Program Files");
+        String inputPath ;
+        Scanner sc = new Scanner(System.in);
+        while(true){
+            int flag =0;
+            System.out.println("ENTER THE NAME OF FILE OR FOLDER");
```

Comment:
are you trying to fetch only files? or folder as well?

Reply:
Yes I am trying to fetch folder also

Reply:
ok